amf-client/shared/src/test/scala/amf/cycle/OasRecursiveFilesCycleTest.scala

```diff
@@ -43,7 +42,7 @@ class OasRecursiveFilesCycleTest extends FunSuiteCycleTests {
 
       RuntimeCompiler
         .forContext(
-          compilerContextBuilder.withEnvironment(Environment.apply()).build(),
+          compilerContextBuilder.build(),
```

Comment:
Can we break this interface?

Reply:
This interface has not been broken (CompilerContextBuilder), removed this code as it was useless.